[2025-02-18T11:20:35.305+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T11:20:35.321+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: postgres_example_v2.create_table scheduled__2025-02-17T00:00:00+00:00 [queued]>
[2025-02-18T11:20:35.326+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: postgres_example_v2.create_table scheduled__2025-02-17T00:00:00+00:00 [queued]>
[2025-02-18T11:20:35.326+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-02-18T11:20:35.333+0000] {taskinstance.py:2889} INFO - Executing <Task(PostgresOperator): create_table> on 2025-02-17 00:00:00+00:00
[2025-02-18T11:20:35.339+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=903) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-18T11:20:35.339+0000] {standard_task_runner.py:72} INFO - Started process 904 to run task
[2025-02-18T11:20:35.339+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'postgres_example_v2', 'create_table', 'scheduled__2025-02-17T00:00:00+00:00', '--job-id', '544', '--raw', '--subdir', 'DAGS_FOLDER/conpostgres.py', '--cfg-path', '/tmp/tmp814bh8ae']
[2025-02-18T11:20:35.341+0000] {standard_task_runner.py:105} INFO - Job 544: Subtask create_table
[2025-02-18T11:20:35.374+0000] {task_command.py:467} INFO - Running <TaskInstance: postgres_example_v2.create_table scheduled__2025-02-17T00:00:00+00:00 [running]> on host 063e621c401a
[2025-02-18T11:20:35.440+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sandeep' AIRFLOW_CTX_DAG_ID='postgres_example_v2' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2025-02-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-17T00:00:00+00:00'
[2025-02-18T11:20:35.441+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-18T11:20:35.455+0000] {sql.py:278} INFO - Executing: 
        CREATE TABLE IF NOT EXISTS mt24061_example_table (
            id SERIAL PRIMARY KEY,
            name varchar NOT NULL
        );
        
[2025-02-18T11:20:35.463+0000] {base.py:84} INFO - Retrieving connection 'postgres'
[2025-02-18T11:20:35.474+0000] {base.py:84} INFO - Retrieving connection 'postgres'
[2025-02-18T11:20:35.584+0000] {sql.py:553} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS mt24061_example_table (
            id SERIAL PRIMARY KEY,
            name varchar NOT NULL
        );
        , parameters: None
[2025-02-18T11:20:35.702+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-18T11:20:35.702+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=postgres_example_v2, task_id=create_table, run_id=scheduled__2025-02-17T00:00:00+00:00, execution_date=20250217T000000, start_date=20250218T112035, end_date=20250218T112035
[2025-02-18T11:20:35.755+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T11:20:35.780+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-18T11:20:35.781+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
